rasa_nlu/evaluate.py

```diff
@@ -638,21 +634,22 @@ def compute_entity_metrics(interpreter, corpus):
     """Computes entity evaluation metrics for a given corpus and
     returns the results
     """
+    entity_results = defaultdict(lambda: defaultdict(list))
+    extractors = get_entity_extractors(interpreter)
+
+    if not extractors:
```

Comment:
again, what if duckling is the only extractor present?

Reply:
hm wait i see your point here, removing those extractors should happen first - it doesn't break anything, but makes more sense to skip everything else.